bin/rsync-solrcloud-backups

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+unset SSH_AUTH_SOCK
+
+set -u
+
+DESTINATION="$1"
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_RSYNC_SOLRCLOUD_BACKUPS_DIR[0]}")/../" && pwd)
```

Comment:
I think this was intended
```suggestion
MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
```

Reply:
Oops, that is most probably due to an hasty search & replace. Fixed in https://github.com/metabrainz/musicbrainz-server/pull/1407/commits/3c0ba68b0422b5b0056b90f89eed42db9cf6e8fd